docs/components/guide-drawer.md

```diff
@@ -0,0 +1,48 @@
+# Guide Drawer
+
+The Guide Drawer helps a user learn about the product by letting them use the real UI.
+
+## Visual Examples
+| Default | Adding Options | Adding Media |
+| --- |---| --- |
+| ![](https://drive.google.com/uc?id=19uRGfYFWxSn7sRrlVIAXy5MsV7fcucQ1) | ![](https://drive.google.com/uc?id=1tUvdz24bEEJalNCHKez3SyIW6dO5H-kL) | ![](https://drive.google.com/uc?id=1lS44XB8knV1ORL7dkcas8_02VA02M-eJ) |
+| Add options to the drawer so that the user can choose where to go next. | A typical guide step includes instructive copy telling the user how to use the normal product interface and can include external links to more information. | Drawer content can include things like images, illustrations or videos. |
+
+## Best Practices
+
+### Guide Drawers should...:
```

Comment:
Is it possible to simplify these section titles so that for each subcomponent/topic the bullets aren't coupled to the title? It's possible I'm reading into the direction of all these pages but I'm just looking for a more plain way forward which gives flexibility in adding notes.

`### Guide Drawers should...:` becomes `### Drawers`

I think then it'd also be nice to get away from the emojis for each bullet. It feels like a heavy overheard to set the tone that they be included throughout these guides. Maybe a simple solution for that is to move the emoji above the list of Dos and the same for the Don'ts so that's there's only one.

Ok, so with that said, I'm perfectly happy to have added this note for discussion, and then to move forward to see what shape further component pages need to take! Obviously, we're prototyping!

Reply:
Fixed! I liked the emojis, but keeping it simple does make it easier to make the tip readable. If we start to feel like the lists are being ignored or are just boring to read, we can think about whether to bring them back.